release-please-config.json

```diff
@@ -13,11 +12,11 @@
     "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
-    "packages/sdk-codegen": { "release-as": "21.0.21" },
-    "packages/sdk-codegen-scripts": { "release-as": "21.0.22" },
+    "packages/sdk-codegen": { "release-as": "21.0.22" },
+    "packages/sdk-codegen-scripts": { "release-as": "21.0.23" },
     "packages/sdk-codegen-utils": { "release-as": "21.0.12"},
     "packages/sdk-node": { },
-    "packages/sdk-rtl": { "release-as": "21.0.17" },
+    "packages/sdk-rtl": { "release-as": "21.0.18" },
```

Comment:
I always set `packages/sdk-*` to version in `.release-please-manifest.json` + 1-patch. Eventually we'll be able to [configure them to always only patch bump](https://github.com/googleapis/release-please/pull/903)  and/or maybe we'll rename/publish them and let them do normal semver